Tools/Microsoft.WinGet.Source/Microsoft.WinGet.Source/Models/DefaultLocaleManifestModels.cs

```diff
@@ -0,0 +1,230 @@
+//----------------------
+// <auto-generated>
+//     Generated using the NJsonSchema v10.4.3.0 (Newtonsoft.Json v11.0.0.0) (http://NJsonSchema.org)
+// </auto-generated>
```

Comment:
These are similar to those found in src/WinGet.RestSource.Utils/Models

Can these replace those? Can we make some minor changes to have a single model source for both?